trackingfront.php

```diff
@@ -81,13 +81,13 @@ public function postProcess()
 			$login = trim(Tools::getValue('login'));
 			$passwd = trim(Tools::getValue('passwd'));
 			if (empty($login))
-				$errors[] = $this->l('login is required');
+				$errors[] = $this->trans('login is required', array(), 'Admin.Global');
 			elseif (!Validate::isGenericName($login))
-				$errors[] = $this->l('invalid login');
+				$errors[] = $this->trans('invalid login');
 			elseif (empty($passwd))
-				$errors[] = $this->l('password is required');
+				$errors[] = $this->trans('password is required', array(), 'Admin.Global');
 			elseif (!Validate::isPasswd($passwd, 1))
-				$errors[] = $this->l('invalid password');
+				$errors[] = $this->trans('invalid password', array(), 'Admin.Global');
```

Comment:
('Invalid password', array(), 'Modules.Trackingfront.Shop')

This string is available only in this module so no need for a global domain.
And if used somewhere else in the back office, should be Admin.Notifications.Error, or if front content Shop.Notifications.Error